app/uk/gov/hmrc/incometaxsubscriptionselfemployedfrontend/views/check_your_answers.scala.html

```diff
@@ -68,6 +68,9 @@ <h2 class="heading-medium">@messages("check_your_answers.sub_heading", businessN
         @business.businessName.map(name => businessName(name, business.id))
         @business.businessTradeName.map(trade => businessTrade(trade, business.id))
     </dl>
+    <p style="margin-top:20px;">
```

Comment:
would prefer not to use custom styling like this here, could you wrap the <dl></dl> tag in a div with class="form-group" and see if that has the same effect?